testbed/tests/widgets/test_activityindicator.py

```diff
@@ -68,3 +66,47 @@ async def test_fixed_square_widget_size(widget, probe):
     # Widget hasn't changed size as a result of being made flexible
     assert probe.height == initial_height
     assert probe.width == initial_width
+
+
+async def test_set_hidden(widget, probe):
+    "Hidden Change functions correctly on this widget regardless of started or stopped"
+
+    # Confirm that stopping hides the widget
+    probe.assert_is_hidden(True)
```

Comment:
This test is incorrect.

"Visible" doesn't mean "is the spinning circle visible". It means "is the widget 'displayed' in the layout". The default state of *every* widget is to be "visible".